### PR TITLE
Goals: Schedule include spent value in calculation

### PR DESCRIPTION
**Pull request by shall0pass**
This adds back the functionality, which was inadvertantly removed, that includes the already spent column in the calculation when the template is run.  

Some transactions may be posted to the account prior to running the templates and would result incorrect budgeted amounts.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ on:
   push:
     branches:
       - master
+      - goalsSweep
```
Comment (j-f1) [inline .github/workflows/build.yml:14]: Why this change?

File: upcoming-release-notes/1049.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+category: Bugfix
+authors: [shall0pass]
+---
+
+Bugfix: Goals Schedule - Include spent in calculation
```
Comment (j-f1) [inline upcoming-release-notes/1049.md:6]:
```suggestion
Goals Schedule - Include spent in calculation
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (shall0pass) [reply .github/workflows/build.yml:14]: Ha!  Snuck in from another branch.  Needs to go!

**Review by j-f1 — APPROVED** [review]
Thanks!
